package.json

```
@@ -20,18 +20,23 @@
   "bugs": {
     "url": "https://github.com/DefinitelyTyped/focus-dt/issues"
   },
+  "engines": {
+    "node": "^20.0.0"
```

```suggestion
    "node": ">=20.0.0"
```

(or `>=20`)

I use Node 22 so this would avoid some sort of warning.